docs/remove-a-service.md

```diff
@@ -0,0 +1,82 @@
+# Remove a service
+
+From time to time, either a service (LGSL code) is no longer valid, required or has been removed from the [Electronic Service Delivery (ESD)](https://standards.esd.org.uk) system. When this happens, we need to remove the service from `local-links-manager` and `publisher`.
+
+## Before you start
+
+Before starting the process it is worth gethering a few key pieces of information:
+
+* The Local Government Services List (LGSL) code of the service you are removing
+* The URL or slug of the service
+* The URL that the service will be redirected to (if appropriate)
+* The ESD URL of the service
+
+## 1. Back-up the links
+
+You may want to back-up the links created for the service before deleting the service.
+
+First run the [`export:links:all`](../lib/tasks/export/link_exporter.rake) rake task (via the console or [`jenkins`](#links)) which exports all of the links for all services in `local-links-manager` to the following file `public/data/links_to_services_provided_by_local_authorities.csv`. If you're running this locally, note that this file is ignored by [`.gitignore`](../.gitignore).
+
+```bash
+bundle exec rake export:links:all
+```
+
+Then retrieve the newly created file using `scp-pull`...
+
+```bash
+gds govuk connect scp-pull -e [integration|staging|production] backend source_file destination_file
+```
+
+Apps on `backend` can be found in `/var/apps/APP-NAME`. So, our data file should be available at `/var/apps/local-links-manager/public/data/links_to_services_provided_by_local_authorities.csv`.
+
+For example:
+
+```bash
+gds govuk connect scp-pull -e integration backend /var/apps/local-links-manager/public/data/links_to_services_provided_by_local_authorities.csv ~/Desktop
+```
+
+**Note**: Once stored in a secure location, please remember to delete the downloaded file from your local machine.
+
+## 2. Unpublish or update the service in `publisher`
+
+There are two options for maintaining the service going forward:
+
+1. Update the format of the service so that becomes a standard plain text page - without a start button and local lookup functionality. Then update the page content as appropriate.
+
+2. Un-publish the service and supply a redirect URL.
```

Comment:
What are we supposed to do with the redirect URL if it's provided?

Reply:
I've updated the doc - and removed the redirect URL from the `Before you start` section. We (as developers) don't do anything with this URL and we don't even need to know if it exists or not to complete our tasks. Hope this clarify's things a bit 🤞

Reply:
Just to complicate things a little the [redirect URL is also optional](https://github.com/alphagov/publisher/blob/a20a5872088eec113b16d66585070aeda2611042/app/controllers/editions_controller.rb#L221).